sdk/test-utils/recorder/MIGRATION.md

```diff
@@ -273,6 +273,10 @@ const {
 // plugins - to be removed
       "karma-json-to-file-reporter",
       "karma-json-preprocessor",
+      //  Also remove the devDependencies from package.json
+      //      - "karma-json-preprocessor": "^0.3.3" 
+      //      - "karma-json-to-file-reporter": "^1.0.1",
```

Comment:
Given that this snippet is strictly about `karma.config.js`, can this change be in another snippet/section?